examples/complete/main.tf

```diff
@@ -10,6 +10,35 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
+##############################################################################
+# Secrets Manager Certificate Setup
+##############################################################################
+
```

Comment:
Keeping inline with making examples full end to end, we usually add support to provision secrets manager and the cert engine, but also provide the ability to use existing too. Can we add that support? You will have to locally test once it since the pipeline will use existing SM

Reply:
added sm instance and private cert engine